incubator/distributed-tensorflow/templates/service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
Both statefulsets use this as governing service. Not sure if this is a good idea. It might be better to have separate services per statefulset.

Reply:
Thank you for your suggestions. I will separate the services per statefulset.